zap/src/main/java/org/zaproxy/zap/extension/history/PopupMenuPurgeSites.java

```diff
@@ -59,16 +62,44 @@ public boolean isButtonEnabledForSiteNode(SiteNode sn) {
 
     @Override
     public void performHistoryReferenceActions(List<HistoryReference> hrefs) {
-        if (hrefs.size() > 0) {
-            int result =
-                    View.getSingleton()
-                            .showConfirmDialog(Constant.messages.getString("sites.purge.warning"));
-            if (result != JOptionPane.YES_OPTION) {
-                return;
+        if (!hrefs.isEmpty()) {
```

Comment:
Check empty and return.

Reply:
This was not changed.

Reply:
Doh, I think I only did it in one class 🤦‍♂️